js/members.js

```diff
@@ -58,7 +59,15 @@ function loadSecondYears(secondYears) {
     members['secondYears'] = secondYears.map(secondYear => getHTMLString(secondYear)).join();
 }
 
-// function to highlight selected tab
+function loadFirstAlumni(firstAlumnis) {
```

Comment:
Rename to ```load2018Alumni(2018Alumnis)```

Accordingly rename all the ```firstAlumni``` occurrences to ```2018Alumni```. Writing the code this way will make it easier to expand to more alumni batches in the future.

Reply:
done